examples/vello_editor/src/text.rs

```diff
@@ -42,6 +42,10 @@ impl Editor {
         self.editor.text()
     }
 
+    pub fn cursor_blink(&mut self) {
+        self.cursor_blink = !self.cursor_blink;
```

Comment:
Might be better to handle this with a periodic function, instead of a toggle, but I won't block on this.

Reply:
Do you mean something like this?
```rs
pub fn cursor_blink(&mut self) {
    if let Some(start_time) = self.start_time {
        let elapsed = Instant::now().duration_since(start_time);
        self.cursor_blink = (elapsed.as_millis() / self.blink_period.as_millis()) % 2 == 0;
    }
}
```

Reply:
Yeah something like that, this also makes it easier to implement solid cursor while actively typing, and to use the timers for more things (I want to add long-touch word selection).